stages/stage_0.py

```diff
@@ -9,7 +9,7 @@
 def run_program(
     program,
     args,
-    quote_kw=KEYWORD_TO_ATOM["q"],
+    quote_kw=KEYWORD_TO_ATOM["quote"],
```

Comment:
I see, the quote op-code *is* configurable now too. shouldn't this use `QUOTE_KW` rather than a hard coded "quote"?

Reply:
actually, shouldn't this parameter name be `quote_atom` and shouldn't the default value be `QUOTE_ATOM`?